build-logic/src/main/kotlin/nessie-common-src.gradle.kts

```diff
@@ -178,7 +178,7 @@ class MemoizedGitInfo {
   companion object {
     private fun execProc(rootProject: Project, cmd: String, vararg args: Any): String {
       val buf = ByteArrayOutputStream()
-      rootProject.exec {
+      rootProject.providers.exec {
```

Comment:
Not sure why, but `Project.exec` got deprecated for removal in 9, but doing that via `ProviderFactory.exec` is not :shrug: